spec/helpers/comments_helper_spec.rb

```diff
@@ -28,6 +28,24 @@
         expect(helper.commenter_id_for_css_classes(comment)).to eq(nil)
       end
     end
+
+    context "when comment is deleted" do
+      let(:comment) { create(:comment) }
```

Comment:
You can do `let(:comment) { create(:comment, is_deleted: true) }` here instead of setting it to `true` in the example. The same should work for `hidden_by_admin`.